include/arch/arm/cortex_m/mpu/arm_mpu.h

```diff
@@ -80,7 +80,14 @@ struct arm_mpu_config {
 		.attr = _attr, \
 	}
 
-/* Reference to the MPU configuration */
+/* Reference to the MPU configuration.
+ *
+ * It shall be defined and populated for each SOC (in SOC definition). It
+ * shall hold the build-time configuration information for the fixed MPU
+ * regions that are to be enabled upon Kernel initialization. It does not
+ * intend to hold regions that are (dynamically) modified during run-time
+ * operation.
```

Comment:
Can we also make it a const?

Reply:
@MaureenHelm , I believe we can do that. Then we would need to change this in all SOCs /Boards that define the boot-time MPU configuration. I opened #10320 for that.